_data/handles/Audit_Avengers.json

```diff
@@ -0,0 +1,20 @@
+{
+  "handle": "Audit_Avengers",
+  "link": "https://twitter.com/JP_Courses",
+  "moralisId": "",
+  "showOnLeaderboard": true,
+  "members": [
+    "JP_Courses",
+    "pxng0lin"
+  ],
+  "paymentAddresses": [
+    {
+      "chain": "polygon",
+      "address": "0x82aAE2d95b92dD5576DbC59F455521250bb9396A"
+    },
+    {
+      "chain": "ethereum",
+      "address": "0x82aAE2d95b92dD5576DbC59F455521250bb9396A"
+    }
+  ]
```

Comment:
Were these teams deleted on purpose? Meaning, did the team members _want_ to retire their team? If so, we should set showOnLeaderboard to false and consider clearing out payment addresses.

Reply:
@captainmangoC4 & @Carlos-stackfive - Correct! All 4 of these were teams that requested to have their team deleted. In addition to hiding from the leaderboard, it would be nice if we can clear the members and payment addresses as well (that's part of the standard process that had been discussed with mango quite some time ago).

Reply:
Ok, I will clear out the members as well; though I think leaving the payment addresses in for now might be better so as to retain wallet address data across the board, particularly for OFAC compliance.
In the future when stuff begins to move over to v1, we'll have soft deletes.